packages/client/src/App.sass

```diff
@@ -3,50 +3,96 @@
 @import "@fontsource/dm-mono"
 $navWidth: 260px
 
-// Set your brand colors
-$bodyBg: #162c4e
-$bodyColor: #bbc6ce
-$primaryPurple: #7464F0
-$backgroundPurple: #9B8FF9
-$darkPurple: #000E64
-
-// Update Bulma's global variables
-$grey-dark: $bodyColor
-$grey-darker: $bodyColor
-$grey-light: $bodyColor
-$link: white
-$primary: $primaryPurple
-$light: $backgroundPurple
+/**********************Initial variables - Colors *******************/
+$black: #000000
+$white: #FFFFFF
```

Comment:
This should be by default like this, not sure if it's needed to be redefined?

Reply:
Tried to remove it, but there was a compiling error saying $white and $black is undefined

Reply:
Is it because it's used here?
$text: $black
$text-invert: $white
$link: $black

Reply:
Yes they are used here and some other places.

Reply:
I tried and had no compilation errors -> after removing $black and $white, and all their occurrences changed to not have the variable prefix, it's working well, can you try again?